scio-extra/src/main/scala/com/spotify/scio/extra/sparkey/package.scala

```diff
@@ -178,4 +179,18 @@ package object sparkey {
       context.sideInput(view).getReader
   }
 
+  sealed trait SparkeyWritable[K, V] extends Serializable {
+    def put(w: SparkeyWriter, key: K, value: V)
+  }
+
+  implicit object StringSparkeyWritable extends SparkeyWritable[String, String] {
```

Comment:
make them `implicit val`s instead? Both are interchangeable but there's no `implicit object` anywhere else in the repo so consistency = easy grepping.